### PR TITLE
Introduce a change to start review online

### PR DESCRIPTION
**Pull request by jonashund**
I will try to add further comments/suggestions after creating this pull request.

### CODE REVIEW

**Comment by jonashund** [comment]
Lines 33 - 39 are the only ones I can review/comment on.
Screenshot from 2022-02-18 10-45-10

File: paper/paper.md [hunk 1]
```diff
@@ -58,13 +58,10 @@ sources is aimed at, for validation reasons.
 
 The methods and operators for the description of linear elasticity and material modeling are mathematically simple and can be expressed in a compact manner.
 Due to this relative simplicity, they are usually implemented independently by each scientist for use in their studies.
-Consequentially, unlike in other research field such as physics and computer science, there is no library for the common methods and operators of linear elasticity and modeling to this day.
-Apart from didactic advantages, this procedure causes problems for
-the exchange of research code and
-the reliability of the implementations.
-This presents a major obastacle with regard to the exchange and reliability of research code as well as the interpretation and comparison of results.
+Consequentially, unlike in other research fields such as physics and computer science, there is no library for the common methods and operators of linear elasticity and modeling to this day.
```
Comment (jonashund) [inline paper/paper.md:61]: fixed typo in line 61

File: paper/paper.md [hunk 2]
```diff
@@ -58,13 +58,10 @@ sources is aimed at, for validation reasons.
...
+This presents a major obstacle with regard to the exchange and reliability of research code as well as the interpretation and comparison of results.
```
Comment (jonashund) [inline paper/paper.md:62]: The two sentences read repetitious.

File: paper/paper.md [hunk 3]
```diff
@@ -58,13 +58,10 @@ sources is aimed at, for validation reasons.
...
 
-The main goal of the project `mechkit` is reusable research code that increases the reliability of the research results and
+The main goal of `mechkit` is reusable research code that increases the reliability of results and
```
Comment (jonashund) [inline paper/paper.md:64]: "the project" has not been used in conjunction with `mechkit` before in this publication

File: paper/paper.md [hunk 4]
```diff
@@ -92,8 +89,8 @@ increases drastically with an increasing number of notations.
 Consequentially, for even a small number of different notations, the implementation of all corresponding translation methods is not feasible.
 Therefore, `mechkit` does not necessarily directly translate one notation into another.
 Instead, in the case of the translation of second- and fourth-order tensors,
-`mechkit` determines the shortest path between source and target notation as illustrated in
-the graph of supported notations, see \autoref{fig:stiffness_graph}.
+`mechkit` determines the shortest path between source and target in the set of implemented notations as illustrated in
```
Comment (jonashund) [inline paper/paper.md:92]: This is a really neat feature and we should consider to emphasize this even further.

File: paper/paper.md [hunk 5]
```diff
@@ -33,13 +33,13 @@ bibliography: paper.bib
 The Python package `mechkit` is a toolkit for researchers
 in the field of continuum mechanics and material modeling.
 `mechkit` contains methods and operators
-that simplify common tasks - especially in the area of tensor algebra and tensor notation.
+for common tasks concerning tensor algebra and tensor notation.
 
 In the context of engineering applications in three spatial dimensions, deformations and stresses in solids are
 usually described by second-order tensors.
-As linear mappings between observed deformations and possibly causal stresses,
+Introduced as linear mappings between the second order tensors of observed deformations and causal stresses,
```
Comment (JulianKarlBauer) [inline paper/paper.md:40]:
```suggestion
As linear mappings between observed deformations and possibly causal stresses,
```

File: paper/paper.md (hunk 1 above)
Comment (JulianKarlBauer) [reply paper/paper.md:61]: thx

**Review by JulianKarlBauer — CHANGES_REQUESTED** [review]

**Review by JulianKarlBauer — APPROVED** [review]

**Pull request merged**